packages/hurl_core/src/ast/core.rs

```diff
@@ -700,6 +700,12 @@ pub struct Variable {
     pub source_info: SourceInfo,
 }
 
+/// Check that variable name is not reserved
```

Comment:
I have added the list of reserved variables here
to be both accessible by both the Hurl file and the CLI options parsers w/o changing module visibility.
We might find a better place though.